### PR TITLE
Convert `admin.module.css` to CSS modules

### PR DESCRIPTION
**Pull request by oisincoveney**
Epic #38811

### Description

Converts standard CSS classes to CSS modules within `admin.module.css`. These classes affect the components within the `Admin` section of the app.

### How to verify

_will write down specific areas to look at_

**_Ensure that nothing looks weird (or surprises you), and that things haven't changed in these areas:_**

_General_
- The left panes within the Admin section of metabase
- Inputs for emails, passwords, and text. Make sure that text sizes, inputs, and the like are consistent.

_Route specific_
`/admin/databases`
- Database list
image

`/admin/datamodel/[segments AND metrics]`
- Actions menu on the right side of the row
image
image

`/admin/settings/maps`
- `Add a new map` widget (ignore the red border, this PR didn't create the problem. It'll be fixed in another PR)
image

`/admin/people`
- the list of users

`/admin/people/groups`
- the popover for a row in groups
image

`/admin/settings/general`
- the site url widget 
image

`/admin/troubleshooting/jobs`
- the left pane
- the modal that appears when clicking `View triggers` on a job
image

`/admin/settings/authentication/api-keys`
- the API key table

`/admin/settings/authentication/[ldap AND saml AND jwt]`
- the group schema / mapping tables
image

`/admin/datamodel/database/`
- Almost all of it - play around and see if you can find anything weird.

	
`/admin/tools/model-caching`
- the table

`/admin/settings/public-sharing`
- the table of public links

`/admin/troubleshooting/tasks`
- the table

### CODE REVIEW

File: frontend/src/metabase/admin/databases/components/DatabaseList/DatabaseList.jsx [hunk 1]
```diff
@@ -75,7 +76,7 @@ export default class DatabaseList extends Component {
 
     return (
       <div className={CS.wrapper} data-testid="database-list">
-        <section className="PageHeader px2 clearfix">
+        <section className={cx("clearfix", AdminS.PageHeader, CS.px2)}>
```
Comment (WiNloSt) [inline frontend/src/metabase/admin/databases/components/DatabaseList/DatabaseList.jsx:79]: I'd advice leaving the order as is so it's very easy to compare to the previous PRs.

```suggestion
        <section className={cx(AdminS.PageHeader, CS.px2, "clearfix")}>
```

File: frontend/src/metabase/css/admin.module.css [hunk 2]
```diff
@@ -2,55 +2,67 @@
   --page-header-padding: 2.375rem;
 }
 
-:global(.ContentTable) {
+:global(.ContentTable),
```
[the remark this answers is not in the document]
Comment (oisincoveney) [reply frontend/src/metabase/css/admin.module.css:5]: Ok, sounds good. Wasn't sure if we would keep it for now or not

[the remark this answers is not in the document]
Comment (oisincoveney) [reply frontend/src/metabase/css/admin.module.css:5]: (even though I know it doesn't make sense 😉 )

File: frontend/src/metabase/admin/datamodel/metadata/components/MetadataTableList/MetadataTableList.tsx [hunk 3]
```diff
@@ -193,7 +200,7 @@ const TableBreadcrumbs = ({ schemaId, onBack }: TableBreadcrumbsProps) => {
         <Icon name="chevronleft" size={10} />
         {t`Schemas`}
       </BackIconContainer>
-      <span className="mx1">/</span>
+      <span className="mx1">-</span>
```
[the remark this answers is not in the document]
Comment (oisincoveney) [reply frontend/src/metabase/admin/datamodel/metadata/components/MetadataTableList/MetadataTableList.tsx:203]: honestly, i have absolutely no idea how that happened

File: frontend/src/metabase/admin/people/components/GroupsListing.jsx [hunk 4]
```diff
@@ -197,7 +198,8 @@ function GroupRow({
 }) {
   const colors = getGroupRowColors();
   const backgroundColor = colors[index % colors.length];
-  const showActionsButton = !isDefaultGroup(group) && !isAdminGroup(group);
+  const showActionsButton =
+    true || (!isDefaultGroup(group) && !isAdminGroup(group));
```
[the remark this answers is not in the document]
Comment (oisincoveney) [reply frontend/src/metabase/admin/people/components/GroupsListing.jsx:202]: good catch, thanks

**Review by deniskaber — APPROVED** [review]
Good, thorough work!

Please fix commented e2e tests logic and then LGTM!

**Pull request merged**